desk/app/notify.hoon

```diff
@@ -165,10 +165,12 @@
   ::
   ++  on-init
     :_  this
-    :~  (~(watch-our pass:io /activity) %activity /notifications)
+    :*  (~(watch-our pass:io /activity) %activity /notifications)
         (~(wait pass:io /clear) (add now.bowl clear-interval))
-        [%pass / %agent [our.bowl %notify] %poke %provider-state-message !>(0)]
         [%pass /eyre %arvo %e %connect [~ /apps/groups/~/notify] dap.bowl]
+      ::
+        :: ?.  =(~rivfur-livmet our.bowl)  ~
```

Comment:
did you mean to leave this commented out?

Reply:
Oops, no. Good catch, thanks.